src/common/Header/styled.js

```diff
@@ -0,0 +1,147 @@
+import styled from 'styled-components';
+import { ReactComponent as VideoIcon } from "./VideoIcon.svg";
+import { ReactComponent as SearchIcon } from "./SearchIcon.svg";
+
+export const Wrapper = styled.header`
+    background: ${({ theme }) => theme.color.woodsmoke};
+    height: 94px;
+    display: grid;
+    gap: 80px;
+    grid-template-columns: 1.5fr 1fr 2fr;
+    padding: 35px 16px 35px 16px;
+    align-content: center;
+    color: ${({ theme }) => theme.color.white};
+    width: 100%;
+
+    @media(max-width: ${({ theme }) => theme.breakpoint.mobileMax}px){
+        height: 142px;
+        grid-template-columns: 1fr 1fr;
+        padding: 16px;
+        gap: 19px;
+    }
+`;
+
+export const TitleWrapper = styled.div`
```

Comment:
I think we don't need `justify-self: right;` and `justify-self: left;` in our case.
Also that `margin-top` is redundant. Header's padding should take care of that.

![image](https://user-images.githubusercontent.com/117176251/220738978-174e43cb-6d98-42e4-8f6e-da0bf2d54bd2.png)

How about that?

export const TitleWrapper = styled.div`
  display: flex;
  gap: 12px;
  align-items: center;

  @media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
    gap: 8px;
  }
`;